helm/silence-operator/templates/configmap.yaml

```
@@ -15,7 +15,7 @@ data:
         address: 'http://0.0.0.0:8000'
     service:
       alertmanager:
-        address: http://alertmanager.monitoring:9093
+        address: http://alertmanager-operated.{{ printf "%s-prometheus " .Values.cluster.name }}:9093
```

The space is unnecessary or I'm missing something
```suggestion
        address: http://alertmanager-operated.{{ printf "%s-prometheus" .Values.cluster.name }}:9093
```